### PR TITLE
Added some spanish and french strings

### PR DESCRIPTION
**Pull request by MehdiKhelfi**

### CODE REVIEW

File: common/language.c [hunk 1]
```diff
@@ -827,6 +831,7 @@ const char* const g_strings[StrId_Max][17] =
         STR_DE("Offline, warte auf Netzwerk…"),
         STR_FR("Hors-ligne, en attente d'une connection..."),
         STR_IT("Disconnesso, in attesa della connessione…"),
+        STR_ES("Desconectado, esperando la red..."),
```
Comment (fincs) [inline common/language.c:834]:
```suggestion
        STR_ES("Desconectado, esperando a la red..."),
```

File: common/language.c [hunk 2]
```diff
@@ -387,6 +389,8 @@ const char* const g_strings[StrId_Max][17] =
         STR_FR("Thème par défaut"),
         STR_DE("Standard Theme"),
         STR_IT("Tema di default"),
+        STR_ES("Tema por defecto"),
+
```
Comment (fincs) [inline common/language.c:393]: Remove extra newline

**Comment by MehdiKhelfi** [comment]
I commited your review, thx

File: common/language.c (hunk 2 above)
Comment (fincs) [reply common/language.c:393]: This is still not resolved

**Comment by MehdiKhelfi** [comment]
I removed the extra newline

**Pull request merged**